README.md

```diff
@@ -46,38 +33,68 @@ As a customer
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
+#### Things that could be added:  
 
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
+[ ] ensure an order can't be completed if nothing in the basket  
+[ ] minimum total of £15 for each order  
+[ ] A customer can delete from the order  
 
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
+----
+#### IRB testing
 
-> :warning: **WARNING:** think twice before you push your **mobile number** or **Twilio API Key** to a public space like GitHub :eyes:
->
-> :key: Now is a great time to think about security and how you can keep your private information secret. You might want to explore environment variables.
+require './lib/restaurant.rb'
 
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
+##### create a restaurant instance:  
+sparkleBurgers = Restaurant.new("Sparkle Burgers LTD")
 
+##### view restaurants menu:  
+sparkleBurgers.menu  
+      => [{:"Cheese burger"=>10}, {:"Cheese and bacon burger"=>12}, {:"Pulled pork burger"=>15}, {:Fries=>6}, {:"Chilli fries"=>8}, {:"Onion rings"=>7}, {:"Coke 330ml"=>2},{:"Fanta 330ml"=>2}, {:"Brew Dog Vegabond 4.5%"=>5}]
 
-In code review we'll be hoping to see:
+sparkleBurgers.view_menu  
+      => "Sparkle Burgers LTD's Menu:\n, 1: Cheese burger, £10\n, 2: Cheese and bacon burger, £12\n, 3: Pulled pork burger, £15\n, 4: Fries, £6\n, 5: Chilli fries, £8\n, 6: Onion rings, £7\n, 7: Coke 330ml, £2\n, 8: Fanta 330ml,£2\n, 9: Brew Dog Vegabond 4.5%, £5"
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this at this moment.
+#####  add to order:  
+sparkleBurgers.choose_dish(2)  
```

Comment:
Adding numbers of dishes rather than names can be more convenient. Maybe including the quantity as a second argument could be faster for the user

Reply:
Totally agree here! I would love to improve this so that both a quantity can be added, and also that if the same dish was added again later in the order, it would also just update the quantity rather than add another to the list